src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +131,33 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe(*, num_channels: int):
```

Comment:
```suggestion
def _create_mock_probe(num_channels: int):
```

We don't generally allow arbitrary positional args to functions (the '*' in signature)

We also tend to follow the practice enforced these days by HDMF of always using keyword arguments